dev/Repeater/InspectingDataSource.cpp

```diff
@@ -136,15 +136,14 @@ void InspectingDataSource::UnListenToCollectionChanges()
     if (auto notifyCollection = m_notifyCollectionChanged.safe_get())
     {
         notifyCollection.CollectionChanged(m_eventToken);
-    } 
-    else if (auto observableCollection = m_observableVector.safe_get())
-    {
-        observableCollection.VectorChanged(m_eventToken);
     }
     else if (auto bindableObservableCollection = m_bindableObservableVector.safe_get())
```

Comment:
Thanks! Please ensure that all the tests pass in the PR pipeline

Reply:
We also must not have test coverage for this. @ranjeshj can you help add tests in the NugetRelease test? If it's CX only we have a CX test app there.